addons/uh60_config/config/CfgAnimationSources.hpp

```diff
@@ -232,40 +232,51 @@ class AnimationSources: AnimationSources {
     mass = MASS_SKIS;
   };
   class HH60Flares_show {
-    displayName = "Show HH-60 Flares";
+    //displayName = "Show HH-60 Flares";
     source="user";
     animPeriod=1;
     initPhase=0;
     mass = MASS_HH60FLARES;
   };
   class HH60GRadar_show {
-    displayName = "Show HH-60 Weather Radar";
+    //displayName = "Show HH-60 Weather Radar";
     source="user";
     animPeriod=1;
     initPhase=0;
   };
   class HH60GFlir_show {
-    displayName = "Show HH-60 Flir";
+    //displayName = "Show HH-60 Flir";
     source="user";
     animPeriod=1;
     initPhase=0;
   };
   class MH60MMisc_show {
-    displayName = "Show MH-60M Exterior parts";
+    //displayName = "Show MH-60M Exterior parts";
     source="user";
     animPeriod=1;
     initPhase=0;
     mass = MASS_MH60M_MISC;
   };
   class Fuelprobe_show {
-    displayName = "Show Fuel Probe";
```

Comment:
Is removing this option intentional? I'd think it'd make sense to keep the option to show and hide the fuel probe and ERFS from the MH/Pavehawk versions

Reply:
From my testing, enabling it still enables the option for the UH variants